packages/gatsby/build.js

```diff
@@ -0,0 +1,12 @@
+const execa = require('execa');
+const { remove } = require('fs-extra');
+
+async function main() {
+  await remove('dist');
+  await execa('tsc', [], { stdio: 'inherit' });
```

Comment:
Should we use `ncc` instead?

Reply:
Redwood/Remix/Hydrogen builders all don't. Personally I don't think we should be using ncc on Builders at all anymore.